Changes.md

```diff
@@ -4,6 +4,7 @@
 Fixes
 -----
 
+- PrimitiveInspector : Fixed bug that caused data lists to loose their scroll position during updates.
```

Comment:
loose -> lose

Reply:
Fixed [here](https://github.com/GafferHQ/gaffer/pull/3906/commits/fc3635f07cb16acb648a72448bfe13dd5a5487e7). Sorry about that.